README.md

```diff
@@ -51,12 +51,33 @@ By default, metrics are formatted using `Elixometer.Utils.name_to_exometer/2`.
 This function takes care of composing metric names with prefix, environment and
 the metric type (e.g. `myapp_prefix.dev.timers.request_time`).
 
-This behaviour can be overridden with a custom formatter function, by adding the
-following configuration entry:
+This behaviour can be overridden with a custom formatter module (implementing the
+`Elixometer.Formatter` behaviour) by adding the following configuration entry:
 
 ```elixir
 config :elixometer, Elixometer.Updater,
-  formatter: &MyApp.Metrics.my_custom_formatter/2
+  formatter: MyApp.Formatter
+```
+
+A formatting module implements the `Elixometer.Formatter` behaviour and implements
+a single function, `format` as such:
+
+```elixir
+defmodule MyApp.Formatter do
+  @behaviour Elixometer.Formatter
+
+  def format(metric_type, name) do
+    # do your formatting here
+    # return [String.t()]
```

Comment:
I think this should be a more concrete example instead of just a comment. For example
```elixir
def format(_metric_type, name) do
  # appends a constant prefix to metric names, and replaces dashes
  # with underscores
  ["my_prefix", String.replace(name, "-", "_")]
end
```